source/backend/discovery/src/lib/finalisation.js

```diff
@@ -0,0 +1,13 @@
+const {AWS_ORGANIZATIONS} = require("./constants");
+
+async function finalise(config, apiClient, accountsMap) {
```

Comment:
nit: not sure about the term finalisation for a module name and this function. something more descriptive might be better that conveys what is being done as part of finalisation?

Reply:
It was more just to to have symmetry with the `initialisation`->`initialize` module at the beginning. Could move the function to the `persistance` module and call it `persistAccountData` or something?

Reply:
yeah I prefer persistance/persistAccountData

Reply:
*done